var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -188,6 +188,14 @@ class Openmpi(AutotoolsPackage):
     # The second patch was applied starting version v4.0.0 and backported to
     # v2.x, v3.0.x, and v3.1.x.
     patch('use_mpi_tkr_sizeof/step_2.patch', when='@1.8.4:2.1.3,3:3.0.1')
+    # in order to work around a bug in older gcc versions on x86_64,
```

Comment:
Packages can grow relatively large over time.  Is the information you provide here in the reference below?  Is so, it would be sufficient to summarize the comment in one line and a second that includes the URL to the issue.

For example, something like "This patch fixes an atomic thread fence performance regression.  See <url> for more." spread across two comment lines?

Reply:
Or you could just use a single comment with the URL.

Reply:
Thanks for review comments, required changes are addressed.